ewsonprem_connector.py

```diff
@@ -508,6 +506,23 @@ def _get_domain(self, username, client_req_id):
 
         return phantom.APP_SUCCESS, domain
 
+    def _set_header_for_rest_call(self, config):
+        """This function is used to update the headers with access_token before making REST call."""
+        if self.get_action_identifier() != phantom.ACTION_ID_TEST_ASSET_CONNECTIVITY:
+            resp_json = None
+            if self._state.get("oauth_token", {}):
+                resp_json = self._state.get("oauth_token", {})
+            if self._state.get("oauth_client_token", {}):
+                resp_json = self._state.get("oauth_client_token", {})
+            if resp_json:
+                self._session.auth = OAuth2TokenAuth(resp_json['access_token'], resp_json['token_type'])
+        # self._is_token_test_connectivity variable set for generating the tokens only for once while having
+        # the multiple calls.
```

Comment:
not sure what this comment means. If it's not necessary, can we remove it?

Reply:
Removed it